docs/docusaurus.config.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
+// `@type` JSDoc annotations allow editor autocompletion and type checking
+// (when paired with `@ts-check`).
+// There are various equivalent ways to declare your Docusaurus config.
+// See: https://docusaurus.io/docs/api/docusaurus-config
```

Comment:
I think we should move this to a TS file for consistency with the rest of the repo.

https://docusaurus.io/docs/typescript-support#typing-config